sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -224,6 +224,9 @@ def compute_table_name(row):
 The GEOGRAPHY data type works with Well-Known Text (See
 https://en.wikipedia.org/wiki/Well-known_text) format for reading and writing
 to BigQuery.
+The BYTES data type requires that bytes are encoded using base64 encoding when
```

Comment:
How about the following wording: 
```BigQuery IO requires values of BYTES datatype to be encoded using base64 encoding when writing to BigQuery. When bytes are read from BigQuery they are returned as base64-encoded bytes.``` 
note: looks like in Java bytes are returned as strings, but in Python as bytes, so there is a difference with Java wording.

Reply:
ok